.travis.yml

```diff
@@ -16,4 +16,6 @@ install:
 - bundle install
 
 script:
-- bundle exec jekyll build
+- bundle exec jekyll build 2> error.log
+- cat >&2 error.log
```

Comment:
The `2>` pipes stderr to error.log

The `>&2` pipes stdout to stderr

> I guess what I am getting at is I don't understand what the >&2 is for.

It makes the font of the error message "Yellow" in Travis?